src/services/lnd/index.ts

```diff
@@ -305,28 +305,34 @@ const lookupPaymentByPubkeyAndHash = async ({
       : PaymentStatus.Pending
 
     let paymentLookup: LnPaymentLookup = {
-      amount: toSats(0),
-      createdAt: new Date(0),
+      status,
       confirmedAt: undefined,
+      createdAt: new Date(0),
       destination: "" as Pubkey,
-      roundedUpFee: toSats(0),
+      milliSatsFee: toMilliSatsFromNumber(0),
+      paymentHash: "" as PaymentHash,
       milliSatsAmount: toMilliSatsFromNumber(0),
+      paths: [],
+      paymentRequest: undefined,
+      roundedUpFee: toSats(0),
       secret: "" as PaymentSecret,
-      request: undefined,
-      status,
+      amount: toSats(0),
     }
 
     if (payment) {
       paymentLookup = Object.assign(paymentLookup, {
-        amount: toSats(payment.tokens),
-        createdAt: new Date(payment.created_at),
+        status,
         confirmedAt: payment.confirmed_at ? new Date(payment.confirmed_at) : undefined,
-        destination: payment.destination as Pubkey,
-        request: payment.request,
-        roundedUpFee: toSats(payment.safe_fee),
+        createdAt: new Date(payment.created_at),
+        destination: payment.destination,
+        milliSatsFee: toMilliSatsFromString(payment.fee_mtokens),
+        paymentHash: payment.id,
         milliSatsAmount: toMilliSatsFromString(payment.mtokens),
-        secret: payment.secret as PaymentSecret,
-        status,
+        paths: payment.paths || [],
+        paymentRequest: payment.request,
```

Comment:
lightning library is not returning `payment.request`, you mentioned that there was a pending library update but I dont see updates to package or yarn.lock

Reply:
This always comes back as `null` so far but the property exists in the `getPayment` type ([see here](https://github.com/alexbosworth/lightning/blob/7ca4a1116e74f1bd5d6b233a066fe6153953c670/lnd_methods/offchain/get_payment.d.ts#L96)) so I assumed it returns sometime. But this makes sense if it doesn't ever return and the library type is somehow wrong right now?

Reply:
Looks like it doesn't return this property when using the `getPayment` method: https://github.com/alexbosworth/lightning/issues/50#issuecomment-965593923